datacapturing/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,10 @@
 
     <uses-permission android:name="android.permission.WAKE_LOCK" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.WRITE_SYNC_SETTINGS" />
```

Comment:
I personally prefer to annotate the requested permissions with the reason, why they were added. This helps when refactoring / cleaning up the code later on.